packages/opencensus-exporter-stackdriver/test/test-stackdriver-cloudtrace.ts

```diff
@@ -273,16 +266,13 @@ describe('Stackdriver Trace Exporter', function() {
             });
 
             nocks.oauth2(body => true);
-            nocks.batchWrite(
-              PROJECT_ID,
-              (body: { spans: Span[] }): boolean => {
-                assert.strictEqual(body.spans.length, 2);
-                const spans = body.spans;
-                assert.strictEqual(spans[0].spanId, rootSpan.id);
-                assert.strictEqual(spans[1].spanId, span.id);
-                return true;
-              }
-            );
+            nocks.batchWrite(PROJECT_ID, (body: { spans: Span[] }): boolean => {
```

Comment:
Is this kind of churn due to a style-formatter upgrade?

Reply:
Yes, there was a security vulnerability in the `gts` formatter